package.yaml

```diff
@@ -24,6 +23,9 @@ ghc-options:
   - -Wno-safe
 
 when:
+  - condition: "impl(ghc >= 9.2)"
```

Comment:
Should this be a template change too?

Reply:
Yeah, I made a few updates to the template after using it and finding issues. This fix was included in [the PR](https://github.com/freckle/haskell-library-template/pull/8).

Reply:
Thank you!